tests/integration/large_message_s3_test.py

```diff
@@ -0,0 +1,76 @@
+import pytest
+import requests
+from urllib.parse import urljoin
+import boto3
+from faust_large_message_serializer.serializer import LargeMessageSerializer
+
+from faust_large_message_serializer.config import LargeMessageSerializerConfig, URIParser
+from urllib3.util.retry import Retry
+from requests.adapters import HTTPAdapter
+
+pytest_plugins = ["docker_compose"]
+
+bucket_name = "my-test-bucket"
+base_path = f"s3://{bucket_name}"
+fake_secret = "fake_secret"
+fake_key = "fake_key"
+endpoint_url = "http://127.0.0.1:4566"
+region = "us-east-1"
+config = LargeMessageSerializerConfig(
+    base_path,
+    0,
+    fake_secret,
+    fake_key,
+    region,
+    endpoint_url,
+)
+
+output_topic = "test-serializer"
+serializer = LargeMessageSerializer(output_topic, config)
+
+
+# Invoking this fixture: 'function_scoped_container_getter' starts all services
+@pytest.fixture(scope="module")
+def wait_for_api(module_scoped_container_getter):
+    """Wait for the api from my_api_service to become responsive"""
+    request_session = requests.Session()
+    retries = Retry(total=20, backoff_factor=1, status_forcelist=[500, 502, 503, 504])
+    request_session.mount("http://", HTTPAdapter(max_retries=retries))
+
+    service = list(
+        filter(
+            lambda x: x.container_port == "4566/tcp",
+            module_scoped_container_getter.get("localstack").network_info,
+        )
+    )[0]
+    hostname = service.hostname if service.hostname != "0.0.0.0" else "127.0.0.1"
+    api_url = "http://%s:%s/" % (hostname, service.host_port)
+    result = request_session.get(api_url).json()
+    assert result["status"] == "running"
+    return request_session, api_url
+
+
+@pytest.fixture(scope="function")
+def s3_bucket_name():
```

Comment:
What is the purpose of the test?

Reply:
it is not a test, it is a fixture. It creates the bucket and delete it when the test is done. it is like the before or after in junit